Misc Lock 11 + auth0.js 9 updates

jeffreylees opened the pull request:
<!---
Notes:
- Your PR should conform to our [Contributing Guidelines](https://github.com/auth0/docs/blob/master/CONTRIBUTING.md)
- If applicable, add details to the [update feed](https://github.com/auth0/docs/tree/master/updates)
- Make sure your PR gets deployed to a Heroku [Review App](https://github.com/auth0/docs/blob/master/CONTRIBUTING.md#review-apps) without errors.
- Please include a short description
- If your PR is a work in progress title it [DO NOT MERGE]
--->

Commit:
Lock + Auth0.js Updates 

Round 1, urgent / incorrect items updates. We will need a more exhaustive set of updates later.

Commit:
Added note to HLP login page

Commit:
Removing a step

Commit:
Adding an auth0.js 9 version to passwordless docs

Mostly just for the CDN URL and to prevent questions, as nothing about implementation of this is changing at the moment.

Commit:
Version URL updates

Commit:
Adding note about COA callback

Commit:
Updating Lock 10 auth params guide to remove warning about scope param

Commit:
Updating with COA doc changes


Removing merge conflicts

aaguiarz committed:
Update _login.md

Updated Cross Origin section

aaguiarz committed:
Update _login.md

aaguiarz commented on articles/architecture-scenarios/implementations/spa-api/spa-implementation-angular2.md:
```diff
@@ -472,7 +472,11 @@ And add a file called `silent.html`:
   <script src="${auth0js_url}"></script>
+=======
+  <script src="${auth0js_urlv9}"></script>
+>>>>>>> Version URL updates
```
Please fix this. This example uses the silent url callback page so it needs to be updated

aaguiarz commented on articles/client-auth/current/client-side-web.md:
```diff
@@ -106,7 +106,11 @@ The [Auth0.js library](https://auth0.com/docs/libraries/auth0js) can assist you
+=======
+  <script src="${auth0js_urlv9}"></script>
+>>>>>>> Version URL updates
   <script>
```
There looks to be a merging issue here

jeffreylees replied:
Ugh, I'll take a look

Commit:
Fixing merge issues

aaguiarz committed:
Update spa-implementation-angular2.md

Removed the section that talked about the callback page for silent authentication

aaguiarz committed:
Links to relevant articles

Links to Custom Domains and Centralized vs Embedded pages.

aaguiarz committed:
Update index.md

Removed mention saying auth0.js v8 was the latest one.

aaguiarz committed:
Update configuration.md

Added rememberLastLogin back

aaguiarz committed:
Update dynamic-client-registration.md

Added rememberLastLogin back

aaguiarz committed:
Update configuration.md

Added remember last login back

aaguiarz committed:
Update user-initiated-linking.md

Added rememberLastLogin back

aaguiarz committed:
Update user-initiated-linking.md

Added rememberLastLogin back
